src/main/scala/dev/atedeg/Table.scala

```diff
@@ -0,0 +1,71 @@
+package dev.atedeg
+
+import java.util.Locale.UK
+
+import scala.util.Try
+
+import dev.atedeg.HtmlParsing.{ extractClassLike, extractNonClassLike }
+
+import better.files.File
+import cats.implicits.*
+import net.steppschuh.markdowngenerator.table.Table.ALIGN_LEFT
+import net.steppschuh.markdowngenerator.table.Table.Builder
+
+import Extensions.*
+
+final case class Row(term: String, definition: String)
```

Comment:
Could you make the constructor private, given that you already have the `buildRow` factory method?